airbyte-ci/connectors/pipelines/pyproject.toml

```diff
@@ -18,6 +18,9 @@ more-itertools = "^8.11.0"
 docker = "^7"
 semver = "^3"
 airbyte-protocol-models = "*"
+# In an ideal world, we would import the version of airbyte-ci for the connector airbyte-ci will interact with
+# Also note that the version that this will currently import is <2 because of the version clash with pydantic
+airbyte-cdk = "*"
```

Comment:
Just a comment to highlight this comment

Reply:
This has been moved outside of the pipelines and therefore is not relevant anymore